package.json

```diff
@@ -45,7 +45,7 @@
     "change": "beachball change",
     "checkchange": "beachball check  --scope \"!sites/*\" --changehint \"Run 'yarn change' to generate a change file\"",
     "check": "beachball check ",
-    "publish": "beachball publish",
+    "publish": "beachball --no-publish --no-git-tags --prerelease-prefix beta",
```

Comment:
what's the command here? the default command is "change"

Reply:
This is just the local script command used for testing.  There's another not shown `publish-ci` that does the heavy lifting. Once this is validated by tested I'll update `publish` and `publish-ci` commands. I hope this answers your question.